website/server/libs/analyticsService.js

```diff
@@ -66,6 +66,10 @@ let _formatUserData = (user) => {
     properties.Mana = Math.floor(user.stats.mp);
   }
 
+  if (user.balance) {
```

Comment:
User.balance should Always exist. Also this check will Always returns false for a balsnce of 0 but we want to store it even if 0